wethegit/corgi
# Fix [project template] ensure .gitignore is present

# What does this solve?
This PR fixes the issue of the `.gitignore` file not being present in a fresh project.

## What was changed?
- Removed custom `.npmignore`
- Rename project template's `.gitignore` to `__.gitignore`
- Add a renaming step to the CLI's `project` command, which renames this file after pulling it down

### Notes
- According to the NPM docs, adding an empty or custom `.npmignore` file should allow us to preserve `.gitignore` files in our shipped packages. Clearly, this was not working, so I opted for a different solution (above)
- Fixes #110 

## Testing this change
- pull the `bugfix/no-gitignore` branch down
- from within the repo's root directory, run the following command: `node ./bin/index.js project testing-gitignore`
- Run `cd testing-gitignore`, and notice that the `.gitignore` file is now present.
